markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+      \end{enumerate}
+  \item Repository: Harvard, ICPSR- Type empty - URL, Journal webpage
+    \begin{lstlisting}[breaklines]
+        AEA Data and Code Repository
+        https://www.openicpsr.org/openicpsr/aea
+    \end{lstlisting}
+  \item Reproducibility:
+      \begin{enumerate}
+            \item Technology environment: empty
+            \item Technology Requirements: Standard, Cluster...; include runtime if available
+            \item Reproduction instructions: No details:
+                \begin{lstlisting}[breaklines]
+                 A readme file with detailed instructions is part of the (external) reproducibility package.
+                \end{lstlisting}
+                 Short details:
+                  \begin{lstlisting}
+                    A readme file and a single do file, AP_final_AEJ. do are provided. 
+                  \end{lstlisting}
+        \end{enumerate}
+\end{enumerate}
+
+## Access and Rights
+\begin{enumerate}
+    \item Disclaimer: \textcolor{red}{Perhaps not necessary}
+      \begin{lstlisting}[breaklines]
+        The views expressed here are the authors, and do not necessarily reflect the official views of the World Bank, its Executive Directors, or the countries they represent.
+      \end{lstlisting}
+    \item Confidentiality: YES or NO, or PARTIAL. \textcolor{red}{Do not add description}
+    \item Citation requirement: (empty)
+    \item License: as in scripts
+        \begin{itemize}
+              \item PP
+              \begin{itemize}
+                \item ICPSR:
+                  \begin{lstlisting}[breaklines]
+                  Creative Commons Attribution 4.0 International (CC BY 4.0) License
+                  https://creativecommons.org/licenses/by/4.0/
+                  \end{lstlisting}
+              \end{itemize}
+              \item WP: 
+              \begin{lstlisting}
+                Modified BSD3
+                https://opensource.org/license/bsd-3-clause/
+              \end{lstlisting}
+            \end{itemize}
+    \item Copyright: (empty)
+\end{enumerate}
+
+## Contacts
+\begin{enumerate}
+    \item WB author or corresponding author: Fill fields Name, email, affiliation, role Author
+      \begin{lstlisting}
+      Patty Smith
+      Author
+      psmith@worldbank.org
+      World Bank
+      \end{lstlisting}
+    \item Reproducibility group contact
+      \begin{lstlisting}
+        Reproducibility WBG
+        reproducibility@worldbank.org
+        reproducibility.worldbank.org
+      \end{lstlisting}
+\end{enumerate}
+
+# Tags
+\begin{enumerate}
+  \item Tag: Partially Reproducible, Fully Reproducible, Not Reproducible, Tag group Reproducible
+  \item Type: PRWP, Published Paper
+\end{enumerate}
+
+# External Resources
+\begin{enumerate}
+  \item Zip file/link for PP and WP
+  \begin{enumerate}
+    \item File type: prg
+    \item Title
+      \begin{lstlisting}
+        Reproducibility package (data and code) for TITLE (of paper), PID PrimaryID
```

Comment:
i think confusing to specify "data and code" here. keep the title the same as elsewhere.

Reply:
It is called as such also in outputs